Integrate_Kobiton_Apps_Repository_with_CI_tool/apps-repo-sample/README.md

```diff
@@ -0,0 +1,81 @@
+# GETTING STARTED
```

Comment:
This should stay as DevOps integration, I've created a repo at https://github.com/kobiton/GitHub-Jenkins-KobitonAppsRepo-Appium-CI-CD, can you help submit PR on this instead ? Note: we're in progress to hire some fresher for doing devops integration , this is not new (https://kobiton.atlassian.net/browse/KOB-1962)